docpages/example_programs/misc/collect_reactions.md

```diff
@@ -1,6 +1,6 @@
 \page collecting-reactions Collecting Reactions
 
-D++ comes with many useful helper classes, but amongst these is something called dpp::collector. Collector is a template which can be specialised to automatically collect objects of a pre-determined type from events for a specific interval of time. Once this time period is up, or the class is otherwise signalled, a method is called with the complete set of collected objects.
+D++ comes with many useful helper classes, but amongst these is something called `dpp::collector`. Collector is a template which can be specialised to automatically collect objects of a pre-determined type from events for a specific interval of time. Once this time period is up, or the class is otherwise signalled, a method is called with the complete set of collected objects.
```

Comment:
check that backticks dont break this as in image below. If this is ok, and not broken, please close PR comment as resolved.

![image](https://github.com/brainboxdotcc/DPP/assets/1556794/8d29faec-6725-4ad1-af93-4abd584d6d17)

Reply:
yeah these look ugly, i think. what do you think?
![image](https://github.com/brainboxdotcc/DPP/assets/1556794/5fa4b399-0706-43fd-b13e-adeb9d0637dc)

Reply:
i share this opinion

Reply:
Agreed, doesn't go well on dark mode either (image for reference, first is before which we don't like, second is a fix).
![image](https://github.com/brainboxdotcc/DPP/assets/46899449/ba82d26c-3f1a-4288-a60e-3f02f37ba9b4)

![imagefixed](https://github.com/brainboxdotcc/DPP/assets/46899449/bdb20f81-b84b-4ab2-898f-8009a6cbb12a)